requirements.txt

```diff
@@ -1,42 +1,39 @@
-aniso8601==3.0.2
-awscli==1.15.71
-backports.functools-lru-cache==1.5
-beautifulsoup4==4.6.1
-boto3==1.7.70
-botocore==1.10.70
-certifi==2018.4.16
+aniso8601==8.0.0
+awscli==1.16.266
+beautifulsoup4==4.8.1
+boto3==1.10.2
+botocore==1.13.2
+certifi==2019.9.11
 chardet==3.0.4
-cheroot==6.4.0
-CherryPy==17.0.0
-click==6.7
-docutils==0.14
-Flask==1.0.2
-Flask-Admin==1.5.1
-Flask-RESTful==0.3.6
-Flask-Script==2.0.6
+Click==7.0
+colorama==0.4.1
+docutils==0.15.2
+Flask==1.1.1
+Flask-RESTful==0.3.7
 Flask-WTF==0.14.2
-idna==2.7
-itsdangerous==0.24
-jaraco.functools==1.20
-Jinja2==2.10
-jmespath==0.9.3
-MarkupSafe==1.0
-more-itertools==4.3.0
-numpy==1.15.0
-portend==2.3
-PyGeoj==0.22
-pyshp==1.2.12
-python-dateutil==2.7.3
-pytz==2018.5
-PyYAML==3.13
-raven==6.9.0
-requests==2.19.1
-s3transfer==0.1.13
+idna==2.8
+itsdangerous==1.1.0
+Jinja2==2.10.3
+jmespath==0.9.4
+MarkupSafe==1.1.1
+more-itertools==7.2.0
+numpy==1.17.3
+packaging==19.2
+pyasn1==0.4.7
+PyGeoj==1.0.0
+pyparsing==2.4.2
+python-dateutil==2.8.0
+pytz==2019.3
+PyYAML==5.1.2
+raven==6.10.0
+requests==2.22.0
+rsa==3.5.0
```

Comment:
I ran into a problem installing requirements because [rsa 3.5.0 doesn't exist](https://pypi.org/project/rsa/#history). I suggest we change this to rsa 3.4.2. Here's a PR: https://github.com/hotosm/MapCampaigner/pull/744